quickjs.c

```diff
@@ -18089,7 +18101,9 @@ static JSValue js_async_function_resolve_call(JSContext *ctx,
         /* return value of await */
         s->func_state.frame.cur_sp[-1] = js_dup(arg);
     }
-    js_async_function_resume(ctx, s);
+    if (!js_async_function_resume(ctx, s)) {
+        return JS_EXCEPTION;
+    }
```

Comment:
GH seems to have swallowed my last review comment... I was saying that single-statement if statements (ones without an `else` clause) usually omit the braces:
```suggestion
    if (!js_async_function_resume(ctx, s))
        return JS_EXCEPTION;
```